components/UnsignedVersion.tsx

```diff
@@ -31,7 +31,7 @@ const UnsignedVersion = () => {
           Collaborator {index + 1}: <br />
           Legal Name: {collaborator.legalName} <br />
           Email address: {collaborator.email} <br />
-          Contribution : {collaborator.typeOfcontribution} <br />
+          Contribution : {collaborator.typeOfMasterContribution} <br />
```

Comment:
Why are you hard-coding to only display the `masterContribution`??? We need logic to show `songwriting` contribution too right???
- actual: hard-coded to master
- expected: logic for both contribution types.